checkstyle-suppressions.xml

```diff
@@ -12,4 +12,8 @@
     <suppress checks="HideUtilityClassConstructor" files="."/>
     <suppress checks="VisibilityModifier" files="."/>
     <suppress checks="MagicNumber" files="test/"/>
+    <suppress checks="MagicNumber" files="db/JobDao"/>
+    <suppress checks="MagicNumber" files="core/JobValidator"/>
+    <suppress checks="AvoidStarImport" files="api/JobService"/>
```

Comment:
Disabling these checks are ok for the DAO but I would follow them for these files.